.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Forem DEVcontainer",
+  "dockerComposeFile": "../docker-compose.yml",
+  "service": "devcontainer",
+  "runServices": ["postgres", "redis"],
+  "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+  "remoteEnv": {
+    "LOCAL_WORKSPACE_FOLDER": "${localWorkspaceFolder}",
+    "GIT_EDITOR": "code --wait"
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {
+      "version": "latest",
+      "enableNonRootDocker": "false",
+      "moby": "true"
+    },
+    "ghcr.io/devcontainers/features/common-utils:2": {
+      "installZsh": "true",
+      "configureZshAsDefaultShell": "true"
+    }
+  },
+  "portsAttributes": {
+    "3000": {
+      "onAutoForward": "notify"
+    },
+    "3035": {
+      "onAutoForward": "silent"
```

Comment:
What's the tradeoff of silent here vs notify above? Is there a reason I (as a user) want 3035 to silently enable?

Reply:
3035 is for webpacker's compiled frontend code, which isn't something Eng ever need to access. It's for the web app to access. I kept it as silent so that it doesn't confuse users by prompted two "port is ready!" message.

Reply:
Since the [devcontainers spec](https://containers.dev/implementors/spec/) states that these files are `JSON with Comments` spec, and since VSCode seems to confirm that [by mapping this path structure](https://github.com/microsoft/vscode/issues/139599) to [Microsoft's JSONC](https://github.com/Microsoft/node-jsonc-parser), can we add this as a source comment to avoid future confusion?

(It looks like MSFT's JSONC isn't particularly widespread: it's used by VSCode and DevContainers and not a whole lot else)

Reply:
Added!